### PR TITLE
create wlecome page and style it using flex layout

### LINKED ISSUE #5: Create Welcome Screen
- [ ] create the layout for the welcome screen

- [ ] Style welcome screen

### PR DESCRIPTION
**Pull request by safamohanna95**
- [x] Created HTML Structure:
      

- Create the basic content of the HTML page
 

- [x] Create the general layout

- divide the page into two parts: upper part and lower part. 
- The upper part contains a fruit basket photo and its shadow
- The lower part contains a header, a paragraph, and a button 

- [x] Use a flex layout 
- [x] Styling the HTML elements
- [x] link the button to the home page

relate #5

### CODE REVIEW

**Review by MohammedHasan-99 — DISMISSED** [review]

**Review by Rawanzaqout — DISMISSED** [review]

File: index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="stylesheets/style.cssgi">
```
Comment (sara219) [inline index.html]: there's an error in the style link, you write _(.cssgi)_, fix it!

Screenshot from 2021-06-25 12-46-15

File: stylesheets/style.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
+* {
+    margin: 0%;
```
Comment (sara219) [inline stylesheets/style.css]: `margin: 0` without `%`

File: stylesheets/style.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    flex-direction: column;
+    align-items: center;
+}
+
+.image-container {
+    width: 100%;
+    background-color: #FFA451;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
```
Comment (sara219) [inline stylesheets/style.css:17]: useless property, delete it

**Review by sara219 — CHANGES_REQUESTED** [review]
Use prettier to format the code (HTML & CSS files)
also, add comments to guide the reviewers.

**Review by MohammedHasan-99 — APPROVED** [review]

**Review by Rawanzaqout — APPROVED** [review]

**Review by sara219 — APPROVED** [review]

**Pull request merged**